### PR TITLE
Continue

### PR DESCRIPTION
**Pull request by kahunacohen**
@hellmanj , @frishberg: added continue flag.

### CODE REVIEW

**Comment by dpsfrishberg** [comment]
Looks great to me.

**Comment by kahunacohen** [comment]
Ok actually let me add a few more tests I just thought of. Meanwhile you can use this branch

File: deflake/cli.py [hunk 1]
```diff
@@ -25,6 +25,8 @@ def parse_args():
 
     parser.add_argument("command", type=str, help="The command to de-flake")
 
+    parser.add_argument("--continue", "-o", action="store_true",
+                        help="When set, deflake will continue even if a process returns a non-zero exist status")
```
Comment (hellmanj) [inline deflake/cli.py]: "exit status"

**Comment by hellmanj** [comment]
awesome

**Comment by kahunacohen** [comment]
Gonna fix up the tests a bit more in another PR and add travis support.

**Pull request merged**